integration-tests/chopsticks/src/network/index.ts

```
@@ -10,7 +10,10 @@ import { ChainConfigs } from './types.js'
 function getRequiredEnvVariable(name: string): string {
 	const value = process.env[name]
 	if (value === undefined) {
-		throw new Error(`Environment variable ${name} is not set.`)
+		if (process.env.NODE_ENV === 'test') {
+			console.error(`Error: Environment variable ${name} is not set. Some tests might not work.`)
+		}
+		return ''
```

what's the reason you are no longer throwing here?

---

Yeah, I found it a bit inconvenient to throw an error if the variables are not set. But I think it's better to throw an error and not run into an undefined state: [4807d9e](https://github.com/KILTprotocol/kilt-node/pull/857/commits/4807d9ee08a46a18f5ace5fb49cf1996b0ac3df4)